spring-webflux/src/test/java/org/springframework/web/reactive/protobuf/Msg.java

```diff
@@ -144,8 +144,8 @@ public boolean hasFoo() {
    */
   public String getFoo() {
     Object ref = foo_;
-    if (ref instanceof String) {
-      return (String) ref;
+    if (ref instanceof String stringRef) {
+      return stringRef;
```

Comment:
```java
// Generated by the protocol buffer compiler.  DO NOT EDIT!
// source: sample.proto
```

For future reference, please do not modify generated code since the changes will be lost if the code is regenerated.

In light of that, I'll revert the changes to generated code when I merge the PR.

Reply:
I'm sorry for my carelessness.